flake/dev/generate-all-maintainers.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, ... }:
+{
+  perSystem =
+    { pkgs, config, ... }:
+    {
+      packages.generate-all-maintainers =
```

Comment:
This name is misleading; a package named "generate" sounds like it produces an executable that can be run to generate something.

This package directly _builds_ the all-maintainers list as its output.